### PR TITLE
fix(NET-581): add tenant tier to status api

### PR DESCRIPTION
**Pull request by Aceix**
## Describe your changes
allow ui to know tenant tier so branding can be applied be fore login

## Provide Issue ticket number if applicable/not in title
NET-581

## Checklist before requesting a review
- [ ] My changes affect only 10 files or less.
- [ ] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [ ] Netmaker is awesome.

### CODE REVIEW

**Review by pwilson77 — APPROVED** [review]

File: controllers/server.go [hunk 1]
```diff
@@ -83,17 +83,24 @@ func getStatus(w http.ResponseWriter, r *http.Request) {
 		DB           bool   `json:"db_connected"`
 		Broker       bool   `json:"broker_connected"`
 		LicenseError string `json:"license_error"`
```
Comment (mattkasun) [inline controllers/server.go:85]: Can this be a bool? Then you could set it to servcfg.IsPro

Comment (Aceix) [reply controllers/server.go:85]: done

**Review by mattkasun — APPROVED** [review]

File: controllers/server.go [hunk 2]
```diff
@@ -94,6 +95,7 @@ func getStatus(w http.ResponseWriter, r *http.Request) {
 		DB:           database.IsConnected(),
 		Broker:       mq.IsConnected(),
 		LicenseError: licenseErr,
+		IsEE:         servercfg.IsPro,
```
Comment (abhishek9686) [inline controllers/server.go:98]: Going forward, the plan is to use only `Pro` terminology instead of `EE`

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: controllers/server.go (hunk 2 above)
Comment (Aceix) [reply controllers/server.go]: changed

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**